statsmodels/tsa/exponential_smoothing/ets.py

```diff
@@ -1748,11 +1752,54 @@ def predict(self, start=None, end=None, dynamic=False, index=None):
         y[0:nsmooth] = self.fittedvalues[start_smooth:end_smooth]
 
         # Out of sample/dynamic prediction: forecast
-        y[nsmooth:] = self._forecast(nforecast, start_forecast)
+        y[nsmooth:] = self._forecast(nforecast, anchor_forecast)
 
         # Wrap data / squeeze where appropriate
         return self.model._wrap_data(y, start, end_forecast - 1)
 
+    def get_prediction(self, start=None, end=None, dynamic=False,
+                       index=None, simulate_repetitions=1000,
+                       **simulate_kwargs):
+        """
+        Returns a
```

Comment:
You should not list the return type here.  Describe in a short sentence what this does.